src/components/LoginRegister.js

```diff
@@ -81,7 +103,17 @@ const LoginRegister = ({ setIsLoggedIn, activeOption, setActiveOption }) => {
                 >
                   Don't have an account?
                 </FormHelperText>
+                <FormHelperText
+                  color="red"
+                  textAlign="left"
+                  mt="3"
+                  mb="3"
+                  display={errorVisible ? 'flex' : 'none'}
+                >
+                  Invalid email or password
+                </FormHelperText>
```

Comment:
Instead of setting a CSS property to hide an element, the usual practice is not to render it at all:

```suggestion
                { errorVisible && <FormHelperText
                  color="red"
                  textAlign="left"
                  mt="3"
                  mb="3"
                >
                  Invalid email or password
                </FormHelperText> }
```